docs/INSTALL.rst

```diff
@@ -75,12 +45,47 @@ to
 SELINUX=permissive
 ```
 
-Create User
------------
+User need to choose appropriate values for their environment:
```

Comment:
Section titles should describe what the section is about. Leave the title as "create user"